.github/workflows/ci.yml

```diff
@@ -399,10 +399,17 @@ jobs:
   elasticsearch-integration-tests:
     if: needs.detect-changes.outputs.java-code-changes == 'true'
     needs: [ detect-changes ]
-    name: "[IT] Elasticsearch"
+    name: "${{ matrix.module.maven_module == 'qa/migration-tests' && 'Migration / ' || '' }}[IT] Elasticsearch - ${{ matrix.module.name }}"
```

Comment:
❓ What is the idea of having this prefix `${{ matrix.module.maven_module == 'qa/migration-tests' && 'Migration / ' || '' }}` when there is also the new suffix ` - ${{ matrix.module.name }}` that helps differentiate the two jobs?

I'd drop the prefix since the suffix should be enough to tell apart QA Acceptance from QA Migration tests. Same for OpenSearch.